### PR TITLE
Aj issue5

### PR DESCRIPTION
**Pull request by adiengineer**
Issue #5 almost done

- Required functionality added
- New Model tests added
- A couple of controller tests remain

### CODE REVIEW

File: app/controllers/images_controller.rb [hunk 1]
```diff
@@ -9,6 +9,11 @@ def create
       # save image and redirect
 
       # save method actually saves image object to db
+      # .save saves to db
```
Comment (abineetds) [inline app/controllers/images_controller.rb]: No need for unnecessary comments

File: app/views/images/new.html.erb [hunk 2]
```diff
@@ -1,4 +1,5 @@
 <%= simple_form_for @image do |f| %>
     <%= f.input :image_uri %>
+    <%= f.input :tag_list %>
```
Comment (abineetds) [inline app/views/images/new.html.erb]: In this input, what happens when you submit the form with tags but with the uri being invalid. What does the validation result look like, do the tags still stay comma-separated?

**Comment by adiengineer** [comment]
Added remaining tests. Looking at requested changes.

File: app/controllers/images_controller.rb (hunk 1 above)
Comment (adiengineer) [reply app/controllers/images_controller.rb]: Yeah. Removed in next commit.

File: app/views/images/new.html.erb (hunk 2 above)
Comment (adiengineer) [reply app/views/images/new.html.erb]: The validation message shows 'not valid' which is correct. But the commas in tags disappear. Not sure what that means.

Comment (adiengineer) [reply app/views/images/new.html.erb]: I am actually adding the tags to the img object only if the image object is valid and the image is saved only in that case.

Comment (adiengineer) [reply app/views/images/new.html.erb]: So the bug is restricted to the 'new' view.

**Pull request merged**